ui/src/components/CopyToClipboardTableCell/README.md

```diff
@@ -0,0 +1,3 @@
+```js
+<CopyToClipboardTableCell tooltipTitle="tooltip" textToCopy="copied text" />
+```
```

Comment:
CopyToClipboardTableCell is a bit hard to document without having it in a table first. The styleguide (styleguidist) that we use doesn't allow importing an external component. For now, let's remove the README.md file.

Note to not forget to re-run `yarn generate` from the root directory as the changes to `ui/README.md` and `README.md` won't be necessary anymore.